.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    branches: main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        package:
+          - debug-print
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - id: tag
+        uses: freckle/haskell-tag-action@v1
+        with:
+          package-yaml: ${{ matrix.package }}/package.yaml
+          tag-prefix: ${{ matrix.package }}-v
```

Comment:
Do we think there's a chance this repository would ever contain more packages? This extra directory is always annoying to work with, and these unnecessarily-prefixed tags seem like another little cut caused by the practice.

I might have to start a discussion in Guides about the pre-emptive sub-directory approach.

Reply:
Probably an -orphans package at least

Reply:
But one can always put other packages in other repos, so

Reply:
I'll admit the `-orphans` example is good for this particular package.